$OpenBSD: patch-Makefile,v 1.2 2000/04/26 15:18:32 camield Exp $

Needed for fake installation. This way the installer can easily be
rebuilt with the fake prefix, without rebuilding _everything_.

--- Makefile.orig	Sat Sep  5 04:33:37 1998
+++ Makefile	Wed Apr 26 17:03:32 2000
@@ -237,7 +237,7 @@
 	./compile cdbmss.c
 
 check: \
-it instcheck
+instcheck
 	./instcheck
 
 commands.o: \
@@ -673,7 +673,7 @@
 	rm -f trysysel.o trysysel
 
 setup: \
-it install
+install
 	./install
 
 sgetopt.o: \
